### PR TITLE
[RFC][NI] V2.0.1 - new styles for ember-addon

### PR DESCRIPTION
**Pull request by mmelo**
v2.0.1

Setting up new styles to be used inside an ember-addon.

### CODE REVIEW

File: src/styles/_components/_step-card.scss [hunk 1]
```diff
@@ -75,11 +75,16 @@
...
+        opacity: .3;
+    }
+
     &--passed &__title {
-        font-size: em(19px);
+        font-size: em(17px);
         color: color('text', 'subtitle');
     }
-
+
```
Comment (satcin) [inline src/styles/_components/_step-card.scss]: remove tab

File: src/styles/_components/_animation.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    perspective: 500px;
+    transform: scale3d(0,0,0);
+    will-change: transform;
+    animation: scaleIn $duration $bezier forwards;
+}
+
+@mixin slide-in($duration: $animation-duration, $bezier: $cubic-bezier) {
+    will-change: transform;
+    animation: slideIn $duration $bezier forwards;
+}
```
Comment (satcin) [inline src/styles/_components/_animation.scss]: EOL

File: src/styles/_components/_alerts.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        font-size: em(17px);
+        font-weight: bold;
+        color: color('text', 'title');
+    }
+    &__content {
+        font-size: em(16px);
+        line-height: 18px;
+        color: color('feedback', 'text');
+    }
+}
```
Comment (satcin) [inline src/styles/_components/_alerts.scss]: EOL

File: src/styles/_components/_animation.scss (hunk 2 above)
Comment (mmelo) [inline src/styles/_components/_animation.scss]: done

File: src/styles/_components/_alerts.scss (hunk 3 above)
Comment (mmelo) [inline src/styles/_components/_alerts.scss]: done

**Comment by mmelo** [comment]
@satcin fixed

**Pull request merged**